packages/ui/components/credits/Credits.tsx

```diff
@@ -1,15 +1,17 @@
+import { useSession } from "next-auth/react";
 import Link from "next/link";
 import { useEffect, useState } from "react";
 
 import { COMPANY_NAME, IS_SELF_HOSTED } from "@calcom/lib/constants";
 import pkg from "@calcom/web/package.json";
 
-export const CalComVersion = `v.${pkg.version}-${!IS_SELF_HOSTED ? "h" : "sh"}-${
-  process.env.CALCOM_LICENSE_KEY === "" ? "ee" : "ce"
-}`;
+export const CalComVersion = (licenseKey: boolean | undefined) => {
+  return `v.${pkg.version}-${!IS_SELF_HOSTED ? "h" : "sh"}-${licenseKey ? "ee" : "ce"}`;
+};
```

Comment:
This should come from a public endpoint

Reply:
Is this a blocker? Otherwise I can work on it in a follow up PR